### ISSUE TITLE
Key/Knob settings must load manually every startup

### ISSUE THREAD

**Issue by senepro** [rsjaffe/MIDI2LR#243]
Dear Mates,

Every time I start LR, I must load manually my Key/knob settings from the previously saved XML file. Could you help me please, how can I set to load my settings automatically? I can't find this function. I tried to save my settings to default.xml (what is not recommended in the FAQ), to another filename.xml, to the plugin dir, to the LR dir, to another drive. Nothing worked.

I tried it with version 0.9.10 and version 1.0.0.

Thank you,
John

**Comment by rsjaffe** [comment]
what operating system?
what directory is the plugin in?

**Comment by senepro** [comment]
Hi rsjaffe!

My system configuration is:

Windows 7 Professional 64 bit
Lightroom CC
The plugin directory is: C:\Program Files\Adobe\Adobe Lightroom\Modules\MIDI2LR.lrplugin\

The Windows and the LR are both clear, legal copy.

Regards,
John

**Comment by rsjaffe** [comment]
Please reinstall the plugin in a directory that you can write to without administrator privileges, such as Documents. Installing the plugin in a program directory can cause these problems.

**Comment by senepro** [comment]
Perfect! I reinstalled the plugin to a simple writeable directory outside the Program Files, and move my setting file to an other writeable directory. Finally I changed the Midi2LR Module settings: Set the Profile folder's path to the directory's path where I store my setting file, and after that everything OK! After restart the LR, and also after restart the whole system. Thank you very much rsjaffe!
